### PR TITLE
Use Vogels Module

### PR DESCRIPTION
**Pull request by nkatzman**
This PR refactors the work that @FenrirUnbound  did with calling the AWS module directly and instead uses the `vogels` module which is a nice syntax on top of the AWS module already

This PR just continues to just create the pipelines table

I use some values in the `screwdriver-data-schema` that don't exist in order to propose that we might want to add values about the dynamodb setup that aren't in it already

Since these are AWS specific, we might want to instead move it under: `schema.aws.hashKey`, `schema.aws.tableName`, .etc but we can talk about that

### CODE REVIEW

File: lib/bobby.js [hunk 1]
```diff
@@ -1,28 +1,50 @@
...
+        vogels.AWS.config.update({ region: this.region });
     }
 
     /**
-     * Create a DynamoDB table for pipelines
+     * Setup pipeline model for creation
+     */
+    setupPipelinesTable() {
+        const modelName = 'pipeline';
+        const piplineSchema = dataSchema[modelName];
```
Comment (d2lam) [inline lib/bobby.js]: pipeline*
typical Noah spelling...

File: lib/bobby.js [hunk 2]
```diff
@@ -1,28 +1,50 @@
...
+        const hashKey = piplineSchema.hashKey || 'id';
```
Comment (d2lam) [inline lib/bobby.js]: Are we going to add `hashKey` to data-schema?

Comment (d2lam) [inline lib/bobby.js]: nvm just read your description :p

**Comment by stjohnjohnson** [comment]
👍 
bubbles_found_his_new_line_of_work_challenging

**Comment by tkyi** [comment]
👍 
giphy-5

**Pull request merged**